core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.ts

```diff
@@ -47,19 +50,48 @@ angular.module('oppia').directive('topicsList', [
             $scope, $uibModal, $rootScope, EditableTopicBackendApiService,
             AlertsService, EVENT_TOPICS_AND_SKILLS_DASHBOARD_REINITIALIZED) {
           var ctrl = this;
-          $scope.getTopicEditorUrl = function(topicId) {
+          /**
+           * @param {String} topicId - ID of the topic.
+           * @returns {String} Url of the topic editor with the
+           * topic iD provided in the args.
+           */
+          ctrl.getTopicEditorUrl = function(topicId) {
```

Comment:
Technically, should use UrlInterpolationService here.

Reply:
Ok updated! Done.